### PR TITLE
Add recipe for scanning_drift_corr

### PR DESCRIPTION
**Pull request by ptim0626**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ptim0626** [comment]
not sure why the osx and win build got time-out?

File: recipes/scanning-drift-corr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - matplotlib
```
Comment (chrisburr) [inline recipes/scanning-drift-corr/meta.yaml:24]: https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base
```

File: recipes/scanning-drift-corr/run_test.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+im2 = np.ones((n,n))
+im1[4,6] = 10
+im1[2,3] = 10
+im2[5,2] = 10
+im2[2,4] = 10
+
+sm = sdc.SPmerge01linear(scanAngles, im1, im2)
+sm = sdc.SPmerge02(sm, 8, 8)
+imageFinal, signalArray, densityArray = sdc.SPmerge03(sm)
+
```
Comment (chrisburr) [inline recipes/scanning-drift-corr/run_test.py:17]: It looks like it gets stuck in an infinite loop:
```
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```

Comment (chrisburr) [reply recipes/scanning-drift-corr/run_test.py:18]: Now it's clearer that the package is actually broken on macOS and Windows. Can you open an issue upstream?

As it's `noarch: python` the issue can be ignored for conda-forge as it'll only be built on Linux after this is merged.

**Pull request merged**